### DISCUSSION TITLE
Welcome to LeCashBot Discussions!

### DISCUSSION THREAD

**Discussion by ryan-harrington** [DamienVesper/lecashbot#64] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

### My Introduction
Hello, I am the owner of LeCashBot. I originally started the project as a way to improve my JavaScript skills when I was first learning the language and to provide a way for the [NitroType server](https://discord.gg/MjsFkWN9HD) to get users to be more active and engaged with the bot. I have been programming since late 2017 writing web applications and teaching myself Computer Science.

**Comment by DamienVesper** [comment]
Hello all! I am DamienVesper, a LeCashBot developer. I have been involved with it and TheTypingMatch for a little over a year now, originally reprogramming the bot and building the foundation that it rested on until it was moved to TypeScript a while ago.

I specialize in backend programming, primary language JavaScript, with a secondary of C++.
Catch me around on Discord at `DamienVesper#0001`.
